README.md

```diff
@@ -211,6 +211,26 @@ So the steps to publish and release in different Gradle invocations are:
 
 (in the above example, steps 1 and 2 could be also combined into `./gradlew publishToSonatype closeSonatypeStagingRepository`, to make only the releasing done in a separate step)
 
+### Summary Tasks
+
+If you declare multiple repositories, you get a separate set of tasks for each of the repositories.
+If you for example declared the repositories `sonatype` and `otherNexus`, you get these tasks:
+- `closeSonatypeStagingRepository`
+- `closeOtherNexusStagingRepository`
+- `releaseSonatypeStagingRepository`
+- `releaseOtherNexusStagingRepository`
+- `closeAndReleaseSonatypeStagingRepository`
+- `closeAndReleaseOtherNexusStagingRepository`
+
+For convenience there are also summary tasks generated, that group the tasks for the different repositories, which are
+- `closeStagingRepository`
+- `releaseStagingRepository`
+- `closeAndReleaseStagingRepository`
```

Comment:
since master is at a 2.x stage, why not go with your original observation/idea, @Vampire?
```suggestion
- `closeStagingRepositories`
- `releaseStagingRepositories`
- `closeAndReleaseStagingRepositories`
```

@szpak acceptable?

Reply:
My guess is, that the main use-case is publishing to Maven Central so 95 % of the users will only have one repository. But technically I agree with "...ies" - of course, it was me mourning about it :-D.
I just made the `...y` for the two new ones to be consistent with the existing one and not sneak in a breaking change.

But I follow what you as maintainers want, just tell me how you like it. :-)

Reply:
Interestingly using plural might lead more users to realizing that it's possible to do multiple, and that could lead to them having "snapshot" and "mavenCentral" repositories, but this might be a stretch.

If we go ahead consider `closeAllStagingRepositories` naming as IMO that makes the plural thing valid, even if there's only one.

For the new ones `...y` makes sense, as it always only touches 1 repo. But the new ones, make the old ones stand out more with them not being plural. Your docs really highlight it :)

Reply:
> that could lead to them having "snapshot" and "mavenCentral" repositories, but this might be a stretch.

That would be strange actually imho. Why should one have those both in parallel? You never want a version published to both. You even cannot by version checks of Sonatype iirc.

> If we go ahead consider closeAllStagingRepositories naming as IMO that makes the plural thing valid, even if there's only one.

Imho the "All" does not make it any more valid, neither with only one nor with multiple.
But I don't have strong feelings about that, just tell me what you want. :-)

> For the new ones ...y makes sense, as it always only touches 1 repo.

I'm not sure what you call "new ones".
The new ones introduced here are `closeStagingRepository` and `releaseStagingRepository`, which are exactly the ones that could touch multiple repositories and thus "...y" does not actually makes sense.

> But the new ones, make the old ones stand out more with them not being plural. 

I'm afraid I also did not fully get this sentence. :-/

Reply:
While technically one could have 2+ (release) repositories configured, I also think simultaneously publishing can be more tricky as the operations are not transactional and the most popular (in the context of two repositories) is releasing to Gradle Plugin Portal and Maven Central (where the former is using the other publishing "technology").

I would expect to use `closeStagingRepository` and `releaseStagingRepository` in 95%+ as a shortcut for releasing to just one place. Nevertheless, I like to idea to emphasis for users that actually there could be more and - in the end - I think that "-ies" is more (logically) accurate. It seems that we all like "-ies" and 2.x seems to be a good time to do it.

Reply:
Ok, changed

Reply:
My bad, I probably messed up "old" and "new", anyway, you got the points :) I shouldn't do reviews that late in the evening.

You're right that there's no point publishing a stable and a snapshot at the same time, but I like szpak's point about GPP and MC, even though it's not actually thing (yet?). I have a feeling in the corporate worlds there are some strange setups where they actually publish to multiple repos.